templates/settings-admin.php

```diff
@@ -64,6 +65,8 @@
 		<input name="redirect_uri" type="text" placeholder="<?php p($l->t('Redirection URI')); ?>">
 		<input name="allow_subdomains" id="allow_subdomains" type="checkbox" class="checkbox" value="1"/>
 		<label for="allow_subdomains"><?php p($l->t('Allow subdomains'));?></label>
+		<input name="trusted" id="trusted" type="checkbox" class="checkbox" value="0"/>
```

Comment:
the "allow_subdomains" checkbox has a value of 1... I expect the same behaviour with the "trusted" checkbox